linode/nb/framework_resource_schema.go

```diff
@@ -77,6 +77,7 @@ var frameworkResourceSchema = schema.Schema{
 		"tags": schema.SetAttribute{
 			ElementType: types.StringType,
 			Optional:    true,
+			Computed:    true,
```

Comment:
I think `tags` isn't computed?

Reply:
Nice catch! This was related to the infinite diff mentioned above, but I was able to come up with a better solution using defaults after taking another look at this issue.